modules/labs-react/package.json

```
@@ -48,6 +48,7 @@
     "@workday/canvas-kit-preview-react": "^7.0.7",
     "@workday/canvas-kit-react": "^7.0.7",
     "@workday/canvas-system-icons-web": "^3.0.0",
+    "@workday/design-assets-types": "^0.2.4",
```

This is fine but let's bump it up to be consistent. It resolves to `0.2.8 `anyway.
```suggestion
    "@workday/design-assets-types": "^0.2.8",
```